sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1115,3 +1115,211 @@ def max_deflection(self):
             return (points[deflections.index(max_def)], max_def)
         else:
             return None
+
+
+class Beam_3d(Beam):
+    """
```

Comment:
I dont think you need to explain `what is beam?` here because this has already been explained in `class Beam()`.

Reply:
Ok, I would change it. Btw should `Beam_3d` inherit from `Beam` class? 
The only methods they both share are setter methods for `length` and `elastic_modulus`. Rest all methods are different for both of the classes.

Reply:
I actually think the `Beam` (2D beam) class should inherit from `Beam_3d` class because all 2D beams can be modelled as a 3D beam. We can discuss this possibility in the future when `Beam_3D` is complete. 

For now, keep the `Beam_3d` class inherit from `Beam` class. We could potentially inherit class functions in the future.